pkg/operator/start.go

```diff
@@ -33,6 +34,9 @@ var (
 
 type RunOptions struct {
 	OperatorPVCEnabled              bool
+	MetricsServerEnabled            bool
+	MetricsPort                     int
+	MetricsInterval                 int
```

Comment:
Nit, should be called `MetricsIntervalSeconds` or equivalent, since we have to dig through references to figure out the unit currently